packages/likert-choice/demo/config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+    elements: {
+        'likert-choice': '..'
+    },
+    models: [
+        {
+            element: 'likert-choice',
+            id: '1',
+            mode: 'evaluate',
+            prompt: 'This instance is in gather mode and can be clicked',
+            disabled: false,
+            keyMode: 'letters',
+            choiceMode: 'radio',
+            activeLang: 'en-US',
+            defaultLang: 'en-US',
+            choices: [
+                {
+                    correct: true,
+                    value: 'disagree',
+                    label: [
```

Comment:
this is a UI custom element, there is no need to add multiple languages to it's model. All this is handled by the controller in `pie-elements`. Your model should be a simple as possible for the ui.
so remove `activeLang`, `defaultLang`, `label: [ {lang, label,value}...]`